doc/Advanced.md

```diff
@@ -28,6 +32,36 @@
 
 [← Go back to Modules Development](ModulesDevelopment.md) | [↑ Go to the Table of Content ↑](../README.md) | [Continue to Frequently Asked Questions →](FAQ.md)
 
+
+## Plugging the QC to an existing DPL workflow
+
+Your existing DPL workflow can simply be considered a publisher. Therefore, replace `o2-qc-run-producer` with your own workflow. 
+
+For example, if TPC wants to monitor the output of the workflow `o2-qc-run-tpcpid`, modify the config file to point to the correct data and do : 
```

Comment:
Maybe "For example, if TPC wants to monitor the output `{"TPC", "CLUSTERS"}` of the workflow..."? It could give a wrong impression, that a workflow may have only one output

Reply:
ok